src/main/java/org/sonatype/repository/vgo/internal/util/VgoDataAccess.java

```diff
@@ -86,6 +93,11 @@ public Asset findAsset(final StorageTx tx, final Bucket bucket, final String ass
     return tx.findAssetWithProperty(MetadataNodeEntityAdapter.P_NAME, assetName, bucket);
   }
 
+  public Iterable<Asset> findPackageAssetsForModule(final StorageTx tx, final Repository repository, final String moduleName) {
```

Comment:
These classes/methods can be tested with a DB instance (see https://github.com/sonatype/nexus-internal/blob/f4440b809afd93712a6ed7904a29a730287f43f6/private/plugins/nexus-repository-yum/src/test/java/org/sonatype/nexus/repository/yum/internal/database/YumQueryComponentTest.java#L64)